tox.ini

```diff
@@ -30,5 +31,10 @@ exclude = .tox,.eggs
 show-source = true
 ignore = D100,D101,D102,D103,D104
 
+[testenv:black]
+basepython = python3.6
+deps = black
```

Comment:
Please prefer to centralize all the dependences management in the `test-requirements.txt` file